src/core/json-schema/src/formly-json-schema.service.ts

```diff
@@ -1,15 +1,25 @@
 import { Injectable } from '@angular/core';
 import { FormlyFieldConfig } from '@ngx-formly/core';
 import { JSONSchema7, JSONSchema7TypeName } from 'json-schema';
+import { ɵreverseDeepMerge as reverseDeepMerge } from '@ngx-formly/core';
```

Comment:
was the ``` ɵreverseDeepMerge  ``` done like that on purpose? It looks like an extra character got added here somehow.

Reply:
Yep that's a "convention" the Angular team uses for not exposing these functions externally, but still allowing them to be shared between libs, hiding them behind this special character 😄. Seems strange but is intended

Reply:
Thanks!  I learned something new!!

Reply:
sure 👍. You can see them being used for example [here in the Angular repo](https://github.com/angular/angular/blob/master/packages/core/src/core_private_export.ts) :)